experimental/directline-speech/javascript_nodejs/02.echo-bot/index.js

```diff
@@ -1,68 +1,72 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-const dotenv = require('dotenv');
 const path = require('path');
 const restify = require('restify');
 
 // Import required bot services.
 // See https://aka.ms/bot-services to learn more about the different parts of a bot.
 const { BotFrameworkAdapter } = require('botbuilder');
-const { BotFrameworkStreamingAdapter } = require('botbuilder-streaming-extensions');
 
 // This bot's main dialog.
 const { EchoBot } = require('./bot');
 
-// Import required bot configuration.
+// Note: Ensure you have a .env file and include MicrosoftAppId and MicrosoftAppPassword to use Direct Line Speech
 const ENV_FILE = path.join(__dirname, '.env');
-dotenv.config({ path: ENV_FILE });
+require('dotenv').config({ path: ENV_FILE });
 
-// Create HTTP server
+// Create HTTP server and configure the Server to handle for Upgrade requests.
 const server = restify.createServer({ handleUpgrades: true });
 server.listen(process.env.port || process.env.PORT || 3978, () => {
     console.log(`\n${ server.name } listening to ${ server.url }`);
-    console.log(`\nGet Bot Framework Emulator: https://aka.ms/botframework-emulator`);
-    console.log(`\nTo talk to your bot, open the emulator select "Open Bot"`);
+    console.log('\nGet Bot Framework Emulator: https://aka.ms/botframework-emulator');
+    console.log('\nTo talk to your bot, open the emulator select "Open Bot"');
 });
 
 // Create adapter.
 // See https://aka.ms/about-bot-adapter to learn more about how bots work.
 const adapter = new BotFrameworkAdapter({
     appId: process.env.MicrosoftAppId,
-    appPassword: process.env.MicrosoftAppPassword
+    appPassword: process.env.MicrosoftAppPassword,
+    enableWebSockets: true
 });
 
 // Catch-all for errors.
 adapter.onTurnError = async (context, error) => {
     // This check writes out errors to console log .vs. app insights.
-    console.error(`\n [onTurnError]: ${ error }`);
+    // NOTE: In production environment, you should consider logging this to Azure
+    //       application insights.
```

Comment:
This information is very valuable, but I wonder if we should provide a link so they can go ahead and do it. The downside is that if the docs link gets stale, so will this.